integration-tests/narayana-lra/src/main/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+quarkus.http.test-port=8081
+quarkus.http.host=${HOST:localhost}
+#quarkus.http.host=0.0.0.0
+#%test.quarkus.http.host=0.0.0.0
```

Comment:
Well they should work so I'd left it in so that we don't forget. I still need to talk to the resteasy integration team about the problem. But if you think it's going to be an issue I can remove it.

Reply:
If they are not used I would suggest removing them, we can add them when they add some value but now it only means that anyone reading the code will stop thinking why this is in here and why it is commented.

Reply:
You missed my point but I can remove them.